Add integration docs for APsystems API (#32096)
home-assistant/home-assistant.github.io: source/_integrations/apsystems.markdown

---
title: APsystems EZ1
description: Control and monitor your APsystems EZ1 microinverters locally without the cloud
featured: false
ha_iot_class: Local Polling
ha_config_flow: true
ha_release: 2024.6
ha_category:
  - Energy
ha_domain: apsystems
ha_platforms:
  - sensor
ha_integration_type: device
---

> 

Reviewer: Can you add some more information on what kind of data is shown? kostal plenticore, for example, provides a list of sensors.

> The APsystems EZ1 integration will allow you to read the data from the inverter, disable its output and set the maximum output.

Reviewer: ```suggestion
The **APsystems EZ1** {% term integration %} allows you to read the data from the inverter, disable its output and set the maximum output.
```
For the glossary terms, you can add a tooltip:
![image](https://github.com/home-assistant/home-assistant.io/assets/38767475/a77d054a-8925-4215-89aa-2bbc1312785c)

> The APsystems EZ1 integration will allow you to read the data from the inverter, disable its output and set the maximum output.

Reviewer: read data from which inverter? any specific type?

Author: It's all the data the inverter measures/provides.

Reviewer: What I'm trying to say is, "the inverter" is a bit vague. is it "...read the data from a APsystems EZ1 microinverter"? Does it support all of them? When I google this, it looks like there are different models.

{% include integrations/config_flow.md %}

> Make sure the local API is activated and set to continuously. For that, connect to the inverter via Bluetooth using the app and go to settings -> local mode and set this to continuously.

Reviewer: Should this ideally be done before adding the integration?

Author: Definitely.

Reviewer: Then this sentence should be moved above the config flow, as a precondition.

---
title: APsystems
description: Control and monitor your APsystems EZ1 microinverters locally without the cloud
ha_iot_class: Local Polling
ha_config_flow: true
ha_release: 2024.6
ha_category:
  - Energy
ha_domain: apsystems
ha_platforms:
  - sensor
ha_integration_type: device
---

> The APsystems {% term integration %} allows you to read the data from your EZ1 inverter.

Reviewer: ```suggestion
The APsystems {% term integration %} allows you to read the data from your [APsystems EZ1](https://emea.apsystems.com/diy/ez1/) microinverter.
```
I think it could be helpful to link to the official product page. Please correct the link if you have one in mind that is more suitable.

Author: The link is fine. Any other links would lead to shops, which is probably unwanted.

> The APsystems {% term integration %} allows you to read the data from your [APsystems EZ1](https://emea.apsystems.com/diy/ez1/) microinverter.

Reviewer: ```suggestion
The **APsystems** {% term integration %} allows you to read the data from your [APsystems EZ1](https://emea.apsystems.com/diy/ez1/) microinverter.
```

The following data is provided by the integration:

- Lifetime production (Per input and in total)
- Current production (Per input and in total)
- Today's production (Per input and in total)

> 

Reviewer: ```suggestion

## Prerequisites

```

> Make sure the local API is activated and set to **continuously**. For that, connect to the inverter via Bluetooth using the app and go to **Settings** > **local mode** and set this to **continuously**.

Reviewer: I do not have this app. But reading through this a second time I'm wondering about the casing. Do they start maybe with a capital letter,  **Local mode** and **Continuously**?

Author: I unfortunately cannot check that, but they probably are.

Author: Here's the best I can find, but this is an old screenshot form their docs
![image](https://github.com/home-assistant/home-assistant.io/assets/66328166/acfd2d23-a6f0-4012-b267-214f8d8e56cb)
 And that's how the app looks in German:
https://raw.githubusercontent.com/SonnenladenGmbH/APsystems-EZ1-API-HomeAssistant/main/assets/images/APsystems-Lokale-API-Aktivieren-Schritt4-6.png

Reviewer: Thank you for checking. Looks like they use "heading-style" casing

---

> title: APsystems EZ1

Reviewer: In the code, this integration is called [APSystems](https://github.com/home-assistant/core/pull/114531/files#diff-b5046a6076f032f4a62a5fe1c6bd17f5282a2249e99ba007ec4de67957aef296R3), not APSystems EZ1. The names should be identical.

description: Control and monitor your APsystems EZ1 microinverters locally without the cloud

> featured: false

Reviewer: ```suggestion
```
Featured false does not need to be set. It is the default.

ha_iot_class: Local Polling
ha_config_flow: true
ha_release: 2024.6
ha_category:
  - Energy
ha_domain: apsystems
ha_platforms:
  - sensor

> ha_integration_type: device

Reviewer: ```suggestion
ha_integration_type: device
```
I tried to double-check for the integration type in the manifest, but could not find it. According to the [dev docs](https://developers.home-assistant.io/docs/creating_integration_manifest?_highlight=integration&_highlight=type#integration-type), it should be defined there.

Author: Created a PR: https://github.com/home-assistant/core/pull/117782

---

The APsystems {% term integration %} allows you to read the data from your EZ1 inverter.
The following data is provided by the integration:

- Lifetime production (Per input and in total)
- Current production (Per input and in total)
- Today's production (Per input and in total)

> Make sure the local API is activated and set to continuously. For that, connect to the inverter via Bluetooth using the app and go to **Settings** > **local mode** and set this to **continuously**.

Reviewer: ```suggestion
Make sure the local API is activated and set to **continuously**. For that, connect to the inverter via Bluetooth using the app and go to **Settings** > **local mode** and set this to **continuously**.
```

---
title: APsystems EZ1
description: Control and monitor your APsystems EZ1 microinverters locally without the cloud
featured: false
ha_iot_class: Local Polling
ha_config_flow: true
ha_release: 2024.6
ha_category:
  - Energy
ha_domain: apsystems
ha_platforms:
  - sensor
ha_integration_type: device
---

The APsystems {% term integration %} allows you to read the data from your EZ1 inverter.
<!--, disable its output and set the maximum output. -->
The following data is provided by the integration:

- Lifetime production (Per input and in total)
- Current production (Per input and in total)
- Today's production (Per input and in total)

<!-- 
The following can be adjusted:

- Status (Output can be enabled and disabled)
- Output limit can be set between 30 and 800 watts

-->

> Make sure the local API is activated and set to continuously. For that, connect to the inverter via Bluetooth using the app and go to settings -> local mode and set this to continuously.

Reviewer: ```suggestion
Make sure the local API is activated and set to continuously. For that, connect to the inverter via Bluetooth using the app and go to **Settings** > **local mode** and set this to **continuously**.
```
